iModelJsNodeAddon/IModelJsNative.cpp

```diff
@@ -2807,12 +2807,43 @@ struct NativeGeoServices : BeObjectWrap<NativeGeoServices>
         return results;
         }
 
+    static Napi::Value GetListOfCRS(NapiInfoCR info)
+        {
+        OPTIONAL_ARGUMENT_ANY_OBJ(0, extent, Napi::Object::New(info.Env()));
+
+        DRange2d extentRange;
+        BeJsGeomUtils::DRange2dFromJson(extentRange, extent);
+
+        // When extent parameter is not provided, extentRange is initialized to (0,0,0,0)
+        bool extentIsValid = (extentRange.low.x != 0 || extentRange.low.y != 0 || extentRange.high.x != 0 || extentRange.high.y != 0);

```

Comment:
Try something like this (I didn't try to compile it):

```suggestion
        DRange2d extentRange;
        bool extentIsValid = ARGUMENT_IS_ANY_OBJ(0);
        if (extentIsValid)
            BeJsGeomUtils::DRange2dFromJson(extentRange, info[i].As<Napi::Object>());

```

Reply:
That did the trick. Thx